eng/cake/helpers.cake

```diff
@@ -85,3 +116,12 @@ void SetEnvironmentVariable(string name, string value, bool prepend = false)
 
     Information("Setting environment variable: {0} = '{1}'", name, value);
 }
+
+bool IsTarget(string target) =>
+    Argument<string>("target", "Default").Equals(target, StringComparison.InvariantCultureIgnoreCase);
+
+bool TargetStartsWith(string target) =>
+    Argument<string>("target", "Default").StartsWith(target, StringComparison.InvariantCultureIgnoreCase);
+
+bool ArgumentSetToTrue(string argName) =>
+    Argument<string>(argName, "false").StartsWith("t", StringComparison.InvariantCultureIgnoreCase);
```

Comment:
Why not just do a string comparison? `t` is a bit weird...

Reply:
because I don't want to type the work "true" :-) you can if you want to. Hence the check.

Ideally we could just do a switch with no parameter but I don't think cake can handle this